template.distillery/PROJECT_NAME.sql

```diff
@@ -12,12 +12,14 @@ CREATE TABLE users ( -- DEFAULT
 
 CREATE TABLE emails ( -- DEFAULT
        email text primary key, -- DEFAULT
-       userid bigint NOT NULL references users(userid) -- DEFAULT
+       userid bigint NOT NULL references users(userid), -- DEFAULT
+       is_primary boolean NOT NULL,
+       is_activated boolean NOT NULL
 );
```

Comment:
I don't think `email` should remain a primary key. Otherwise, you can register any email address and make it difficult for the actual owner of the address email to use it.

So, we should have the following constraints:
* activated emails should be unique:
````
CREATE UNIQUE INDEX email_unique ON emails (email) WHERE is_activated;
````
* each user should have at most one primary email:
````
CREATE UNIQUE INDEX user_unique ON emails (email) WHERE is_primary;
````

Or maybe just activated email addresses should be unique, so that we can have recovery addresses shared between several accounts?